ts/Stock/Indicators/RSI/RSIIndicator.ts

```diff
@@ -64,7 +64,8 @@ class RSIIndicator extends SMAIndicator {
     public static defaultOptions: RSIOptions = merge(SMAIndicator.defaultOptions, {
         params: {
             period: 14,
-            decimals: 4
+            decimals: 4,
+            index: 3
```

Comment:
An index declared here is not consistent with the [API](https://api.highcharts.com/highstock/series.rsi.params), which one is correct?

Reply:
Previously, the default index value was hard coded in the getValues function, and wasn't taking into consideration the value passed through series configuration. Because the index value was not defined here, the api specification was inherited from SMA indicator, where default index value was 3. Adding the default index value here not only allows us to change the configuration of the RSI indicator, which allows to calculate the indicator over other values, but also changes the API to the correct value.